astronomer/providers/google/cloud/xcom_backends/gcs.py

```diff
@@ -0,0 +1,67 @@
+import json
+import os
+import uuid
+from typing import Any, Union
+
+import pandas as pd
+from airflow.models.xcom import BaseXCom
+from airflow.providers.google.cloud.hooks.gcs import GCSHook
+
+
+class GCSXComBackend(BaseXCom):
+    """
+    Custom XCom persistence class extends base to support various datatypes.
+    Add the below line in the environment
+    AIRFLOW__CORE__XCOM_BACKEND=
+    astronomer.providers.google.cloud.xcom_backends.gcs.GCSXComBackend
+    """
+
+    PREFIX = os.getenv("PREFIX", "GCSXCOM_")
+    GCP_CONN_ID = os.getenv("CONNECTION_NAME", "google_cloud_default")
+    BUCKET_NAME = os.getenv("XCOM_BACKEND_BUCKET_NAME", "some_bucket_name")
```

Comment:
Either users should be required to set `XCOM_BACKEND_BUCKET_NAME` or the default value should be something more proper IMO.

Reply:
+1